src/slices/directanswer.ts

```diff
@@ -15,6 +15,10 @@ const reducers = {
 
 /**
  * Registers with Redux the slice of {@link State} pertaining to direct answers.
+ *
+ * @param prefix - The prefix to access the part of the Redux store specific to the
+ *                 AnswersHeadless instance
```

Comment:
@oshi97 Can you confirm that this is accurate?

Reply:
Yup!